controllers/clustercurator_controller.go

```diff
@@ -44,6 +57,35 @@ func (r *ClusterCuratorReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, nil
 	}
 
+	// If the ClusterCurator contains at least one Workflow hook type then
+	// check the AnsibleJob CRD to make sure it can support Workflow invocation.
+	if containsWorkflowType(curator) {
+		err := r.checkWorkflowSupport(ctx, curator)
+		if err != nil {
```

Comment:
It seems errors returned by r.checkWorkflowSupport() may have various reasons. WorkflowNotSupported is one of them.